.github/workflows/daily-tests.yaml

```diff
@@ -102,7 +80,7 @@ jobs:
 
             - name: long ${{ matrix.test-type }} tests
               working-directory: ${{ github.workspace }}/tests
-              run: ./main.py run gem5/${{ matrix.test-type }} -j$(nproc) --length=long -vv -t $(nproc)
+              run: ./main.py run gem5/${{ matrix.test-type }} -j$(nproc) --length=long -vv -t $(nproc) --exclude-tags gcn_gpu
```

Comment:
Why would we want to exclude GPU tests here?

Reply:
I think I was asking why the gcn-gpu tests are being excluded (why "run all these tests except those that require gcn-gpu"), not what the flag does.  Maybe unittests-fast-debug is too fast for GPU tests to be included or something?